README.md

```diff
@@ -308,6 +308,46 @@ This should be verified by executing the Pelion E2E python test library tests.
 
 <span class="notes">**Note:** Future version will bring in also firmware update test as part of the minimum test set.</span>.
 
+## Contributing platform support
+
+The contribution of platform support to this repository is restricted to Arm Mbed Partners and Arm Engineering teams. If you’d like to add a custom or community-based platform, please fork this repository and add it into your own account.
+Expectations on contributions:
+
+-	No code changes in `main.cpp`.
+This is a minimal and generic application that’s expected to work on out of the box with all platforms listed in the documentation and [Pelion Quick-start](https://os.mbed.com/guides/connect-device-to-pelion/) guide
+
+- No changes to the hash of `mbed-os.lib ` or `mbed-cloud-client.lib` files.
+These are maintained and updated by the Arm team.
+
+-	No extra files or `.mbedignore` with removal of Mbed OS code.
+
+-	Configuration (required)
+     - `mbed_app.json` to add components or features. Please follow the guidelines in the porting section of the docs.
+
+-	Drivers (optional)
+     -	If required, drivers for networking or storage (non-default) can be added in the `drivers` folder using an external library (.lib). For example  `COMPONENT_MYDRIVER.lib` and enabling in `mbed_app.json`.
+
+-	Bootloader (required)
+     -	The configuration should be provided in either [mbed-bootloader](https://github.com/ARMmbed/mbed-bootloader) repository (as default configuration) or bootloader folder in this repository (if non-default). Our recommendation is to contribute to the mbed-bootloader repository whether possible. Please indicate where the bootloader configuration lives.
+     -	Binaries should be generated and contributed following the name conventions (see bootloader folder).
+
+-	Indication of platform support
+     -	Please update `README.md` file and add an entry to the list of supported boards.
+
+-	Test results and other information
```

Comment:
I think it will be helpful to provide an example template on how to report the results so there can be consistency in reporting.

Reply:
This would be nice. Perhaps we could create a PR template with this information and tick boxes wherever is required?